src/python_testing/TestSpecParsingDeviceType.py

```diff
@@ -46,6 +46,10 @@ def setup_class(self):
         self.classification_scope = "endpoint"
         self.clusters = {0x0003: "Identify", 0x0004: "Groups"}
 
+        # Let's just build a dictionary of device types to IDs becasue I need them and we don't have codegen
+        self.dt_ids = {dt.name.replace(' ', '').replace('/', '').replace('-', '').lower()
```

Comment:
nit:

```suggestion
        self.dt_ids = {re.replace('[ -/]*', '', dt.name.lower())
```

is maybe shorter (assuming re is imported)